### PR TITLE
Clean up fhir-persistence-schematool usage info #3072

### PR DESCRIPTION
**Pull request by prb112**
- Created a simple Enum and Menu System
- Generates Markdown from the Enum 
- Generates PrintStream output from the Enum to print usage

Addresses #3072 

Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/util/CommonUtil.java [hunk 1]
```diff
@@ -38,114 +37,11 @@
...
-        ps.println("--drop-detached tenantName");
-        ps.println(" * (phase 2) drops the detached tenant partition tables given the tenantName");
-
-        // Deletes tenant meta data
-        ps.println("--delete-tenant-meta tenantName");
-        ps.println(" * deletes tenant metadata given the tenantName");
-
-        ps.println("--list-tenants");
-        ps.println(" * fetches list of tenants and current status");
+    public static void printUsage() {
```
Comment (lmsurpre) [inline fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/util/CommonUtil.java:41]: is this method empty now?

File: fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/menu/Menu.java [hunk 2]
```diff
@@ -0,0 +1,176 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.schema.app.menu;
```
Comment (lmsurpre) [inline fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/menu/Menu.java:7]: not sure it needs its own package, but its probably fine

File: fhir-persistence-schema/README.md [hunk 3]
```diff
@@ -772,3 +772,52 @@ logical_resource_security           | Security labels associated with the resour
...
+|--vacuum-cost-limit|costLimit|The Vacuum cost limit to set|
+|--skip-allocate-if-tenant-exists||Skips allocating a tenant if it already exists|
+|--force-unused-table-removal||Forces the removal of unused tables - DomainResource, Resource|
+|--prop|name=value|name=value that is passed in on the commandline|
+|--pool-size|poolSize|poolsize used with the database actions|
+|--drop-schema-oauth||drop the db schema used by liberty's oauth/openid connect features|
+|--drop-schema-batch||drop the db schema used by liberty's java-batch feature"|
+|--drop-schema-fhir||drop the schema set by '--schema-name'|
+|--update-schema||deploy or update the schema set by '--schema-name'|
+|--update-schema-fhir||Updates the FHIR Data Schema|
```
Comment (lmsurpre) [inline fhir-persistence-schema/README.md:817]: this one also applies to the schema set by `--schema-name`, right?
IIRC its basically synonymous with `-update-schema` but we added `--update-schema-fhir` when we added those other schemas for improved consistency/clarity.
Maybe we should mark the `--update-schema` flavor as deprecated?

Comment (prb112) [reply fhir-persistence-schema/README.md:817]: It uses a variable... I've updated the table and the menu with schemaName

I marked it as deprecated

**Review by lmsurpre — APPROVED** [review]
LGTM

**Pull request merged**